src/psbt.js

```diff
@@ -231,6 +234,135 @@ export function psbtOutputFormatter(output) {
   }
 }
 
+/**
+ * Translates a PSBT into inputs/outputs consumable by non-PSBT devices in the
+ * `unchained-wallets` library.
+ *
+ * FIXME - Have only confirmed this is working for P2SH addresses on Ledger on regtest
+ *
+ * @param {module:networks.NETWORKS} network - bitcoin network
+ * @param {module:multisig.MULTISIG_ADDRESS_TYPES} addressType - address type
+ * @param {String} psbt - PSBT as a base64 or hex string
+ * @param {Object} keyDetails - Object containing signing key details (Fingerprint + bip32 prefix)
+ * @returns {null|Object} returns object with the format
```

Comment:
If we were to name the return object what would it be? We have a lot of these different objects with different shapes that are returned in this library, and I'm wondering if there is any other corollary. While we're not going to do it in this PR, Typescript helps here because it forces us to be explicit about the _type_ that is returned and then we can easily keep track (and let our IDEs do so as well)

Reply:
hmm - I don't know what I'd call it. `txIngredients` ? `txDetails` ? `txInfo`? `signingDetails`? We're not really defining anything new here, it's just hey - this is what we already know the devices will accept.

It's all the stuff the nonPSBT device needs to sign the transaction (minus the network).

Reply:
going to go back on what i said here a bit just to say that it's the unchained-specific object that does contain txDetails but in a special way that really only we care about (especially right now)